### PR TITLE
[code-infra] Rename repo to mui/toolpad

### PR DESCRIPTION
**Pull request by Janpot**
To be merged after the repository has bee renamed in GitHub

Closes https://github.com/mui/toolpad/issues/3373

### CODE REVIEW

**Comment by Janpot** [comment]
The repo has been renamed

**Comment by prakhargupta1** [comment]
> The repo has been renamed

Neat!

**Review by apedroferreira — APPROVED** [review]

**Comment by oliviertassinari** [comment]
Nice, 

```js
git remote set-url upstream git@github.com:mui/toolpad.git
```

File: packages/toolpad-core/package.json [hunk 1]
```diff
@@ -29,14 +29,14 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mui/mui-toolpad.git",
+    "url": "https://github.com/mui/toolpad.git",
```
Comment (oliviertassinari) [inline packages/toolpad-core/package.json:32]: Wrong here

```suggestion
    "url": "git+https://github.com/mui/mui-toolpad.git",
```

https://publint.dev/@toolpad/core@0.6.0

Comment (oliviertassinari) [inline packages/toolpad-core/package.json:32]: Wrong here

```suggestion
    "url": "git+https://github.com/mui/toolpad.git",
```

https://publint.dev/@toolpad/core@0.6.0

Comment (Janpot) [reply packages/toolpad-core/package.json:32]: > This looks wrong, no?

But it's not 😄  https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository

> Specify the place where your code lives. This is helpful for people who want to contribute. If the git repo is on GitHub, then the npm repo command will be able to find you.
> 
> Do it like this:
> 
> ```json
> {
>   "repository": {
>     "type": "git",
>     "url": "git+https://github.com/npm/cli.git"
>   }
> }
> ```
> 
> The URL should be a publicly available (perhaps read-only) URL that can be handed directly to a VCS program without any modification. It should not be a URL to an html project page that you put in your browser. It's for computers.

Comment (Janpot) [reply packages/toolpad-core/package.json:32]: Opening https://github.com/mui/toolpad/pull/4113 to correct

**Pull request merged**